utilities/logic/wait_for.py

```diff
@@ -360,6 +365,7 @@ def main():
             delay=dict(default=0, type='int'),
             port=dict(default=None, type='int'),
             path=dict(default=None, type='path'),
+            ignore_time_wait=dict(default=None),
```

Comment:
I'm inclined to think this should be a bool type rather than None vs anything. should be clearer for usage docs and predictability.

Some other examples:

https://github.com/ansible/ansible-modules-core/search?utf8=%E2%9C%93&q=type+%3D%27bool